jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamo/editor/actions/ConfigureMaxResultsAction.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamo.editor.actions
+
+import com.intellij.openapi.actionSystem.ActionManager
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.ComputableActionGroup
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.openapi.actionSystem.PlatformDataKeys
+import com.intellij.openapi.actionSystem.ToggleAction
+import com.intellij.openapi.project.DumbAware
+import software.aws.toolkits.jetbrains.services.dynamo.editor.DynamoTableEditor
+
+class ConfigureMaxResultsAction : ComputableActionGroup.Simple(/* popup */ true) {
+    override fun computeChildren(manager: ActionManager): Array<AnAction> = DynamoTableEditor.MAX_RESULTS_OPTIONS.map { (ChangeMaxResults(it)) }.toTypedArray()
+
+    private class ChangeMaxResults(private val choice: Int) : ToggleAction(choice.toString()), DumbAware {
+        override fun isSelected(e: AnActionEvent): Boolean = getEditorState(e.dataContext)?.maxResults == choice
+
+        override fun setSelected(e: AnActionEvent, state: Boolean) {
+            if (state) {
```

Comment:
nit: state->selected? also we don't need to do anything if we turn it off? Shouldn't we set it to default?

Reply:
You can't "unselect" anything. Toggling 1 "unselects" the other options

Reply:
<img width="230" alt="Screen Shot 2021-05-14 at 1 21 51 PM" src="https://user-images.githubusercontent.com/8992246/118324809-61566980-b4b7-11eb-8f02-578811725b10.png">